### DISCUSSION TITLE
Multi-languge web-site

### DISCUSSION THREAD

**Discussion by HABIBULLO77** [smaranjitghose/awesome-portfolio-websites#1160] [Ideas]
How can this website be a multi-language one? Can you make this?

**Comment by smaranjitghose** [comment]
@HABIBULLO77 
You can do it with a language selector tag and corresponding text in the js files. [Example](https://www.geeksforgeeks.org/how-to-serve-a-page-with-content-in-multiple-languages/).

Thanks for the suggestion. We would put this in the roadmap of the project
